test/Microsoft.OpenApi.Tests/PublicApi/PublicApi.approved.txt

```diff
@@ -179,8 +195,13 @@ namespace Microsoft.OpenApi.Extensions
         public static Json.Schema.JsonSchemaBuilder AdditionalPropertiesAllowed(this Json.Schema.JsonSchemaBuilder builder, bool additionalPropertiesAllowed) { }
         public static Json.Schema.JsonSchemaBuilder Discriminator(this Json.Schema.JsonSchemaBuilder builder, Microsoft.OpenApi.Models.OpenApiDiscriminator discriminator) { }
         public static Json.Schema.JsonSchemaBuilder ExclusiveMaximum(this Json.Schema.JsonSchemaBuilder builder, bool value) { }
+        public static Json.Schema.JsonSchemaBuilder ExclusiveMaximum(this Json.Schema.JsonSchemaBuilder builder, double value) { }
```

Comment:
I'm not sure we should have the double overloads here?

Reply:
We need to define the overloads as extension methods to the JsonSchema builder instance for us to support adding schema values of that type

Reply:
In OpenAPI 3.1, OpenAPI schema is fully compatible with JSON schema hence the `exclusiveMinimum` and `exclusiveMaximum` properties are of type Number and hold distinct values. 
Therefore, they convey the same meaning as `exclusiveMinimum` does in conjunction with `minimum` or `exclusiveMaximum` with `maximum` respectively.
[Reference](https://www.openapis.org/blog/2021/02/16/migrating-from-openapi-3-0-to-3-1-0)